pkg/dogecoin/libdogecoin.go

```diff
@@ -0,0 +1,28 @@
+package dogecoin
+
+import (
+	giga "github.com/dogecoinfoundation/gigawallet/pkg"
+
+	"github.com/jaxlotl/go-libdogecoin"
+)
+
+var _ giga.DogecoinL1 = DogecoinL1Libdogecoin{}
+
+/* Returns a Mocked giga.DogecoinL1 implementor */
```

Comment:
not mocked at all, this is the real deal :D

Reply:
Oh right lol